src/SIE/Data/Verification.php

```diff
@@ -173,8 +173,12 @@ public function addTransaction(Transaction $transaction)
      * Get all transactions
      * @return Transaction[]
      */
-    public function getTransactions()
+    public function getTransactions(): array
```

Comment:
I think the proper fix is to initialize it in the constructor instead, can you do that and revert these changes?

Well spotted!

Reply:
I added both property initialization and constructor initialization
This makes it clear that $transactions is intended to be an array, even without looking at the constructor. But if all you need is constructor initialization I'll remove property init ;)